src/course-outline/messages.js

```diff
@@ -29,6 +29,21 @@ const messages = defineMessages({
     id: 'course-authoring.course-outline.section-list.button.new-section',
     defaultMessage: 'New section',
   },
+  exportTagsCreatingToastMessage: {
+    id: 'course-authoring.course-outline.export-tags.toast.creating.message',
+    defaultMessage: 'Please wait. Creating export file...',
+    description: 'In progress message in toast when exporting tags of a course',
+  },
+  exportTagsSuccessToastMessage: {
+    id: 'course-authoring.course-outline.export-tags.toast.success.message',
+    defaultMessage: 'File created successfully',
```

Comment:
I'm assuming this message will change later, but it's too unspecific. This is specifically a message about tags being exported from a particular course and that being successful, so the message can be more tailored tot hat.

Reply:
Sure :+1: Updated: https://github.com/openedx/frontend-app-course-authoring/pull/995/commits/1d37c848e9cc67c1aac769c946d34924d906e4e6